windows-driver-docs-pr/mobilebroadband/cosa-overview.md

```diff
@@ -28,7 +28,21 @@ These events trigger the Windows provisioning engine to look for a change in set
 
 ## What SIM information from modems does COSA use?
 
-For MO/MVNO discovery, Windows tries to make the best match for an available profile in the COSA database using SPN from the SIM in the modem.
+For MO/MVNO discovery, Windows tries to make the best match for an available profile in the COSA database using using identifiers such as MCC / MNC, ICCID, IMSI, SPN, GID1 from the SIM in the modem.
+
+## What is a COSA profile?
+
+The COSA database consists of profiles. Each profile contains a set of cellular provisioning rules applied to a device, like APN and MO branding information to be displayed in the Windows UX. Each profile also has a set of targets, matching a cellular device to a set of cellular provisioning rules. Each MO has their own COSA profile, but some MOs will have multiple profiles for specific use cases (like implementing a 5G private network or an eSIM provisioning profile).
+
+## How does a COSA profile get matched to a device?
+
+Each COSA profile contains a set of targets used by Windows to match the profile to the device. MOs can set which target identifiers they'd like to use. The target identifiers that can be used are as follows: MNC, MCC, SPN, PNN, GID1, ICCID, IMSI. Windows looks for the profile with the highest number of target identifiers to use. If for instance Windows detects that the device's SIM matches with 1 identifier (SPN) from COSA profile A, but 2 identifiers from COSA profile B (SPN and ICCID), Windows will utilize the COSA profile A.
+
+## Can OEMs customize COSA?
+
+OEMs can extend COSA to meet their needs. Windows will prioritize the OEM customization. For more information on OEM customizations of COSA, see [Customize the Country and Operator Settings Asset](/windows-hardware/customize/desktop/customize-cosa).
+
```

Comment:
We don't want OEMs to customize COSA, right? The preferred path is to update the central database. I think this text should reflect that.

Reply:
We don't, but situations happen where this is wanted. I think we need to keep documentation for this, but I'll update the text here to make it clear that this is not the preferred method.